### PR TITLE
Faraji settings view

### PR DESCRIPTION
**Pull request by Farajialireza82**
created the XML file, fragment, and activity for the settings section

### CODE REVIEW

File: settings/src/main/java/com/islamversity/settings/SettingsView.kt [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package com.islamversity.settings
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import androidx.preference.PreferenceFragmentCompat
+import com.example.search.R
+
+class SettingsView : AppCompatActivity() {
```
Comment (rezaiyan) [inline settings/src/main/java/com/islamversity/settings/SettingsView.kt]: Like rest of views, `SettingsView `must be a extended from [CoroutineView](https://github.com/islamversity/Reyan/blob/ed2e2546748038e15fa3dc014526d7759fe2d1d4/base/src/main/java/com/islamversity/base/CoroutineView.kt)

File: app/src/main/java/com/islamversity/reyan/MainActivity.kt [hunk 2]
```diff
@@ -46,7 +47,7 @@ class MainActivity : AppCompatActivity(),
         setContentView(R.layout.activity_main)
 
         router = Conductor.attachRouter(this, root, savedInstanceState)
-        val view = QuranHomeView()
+        val view = SettingsView()
```
Comment (rezaiyan) [inline app/src/main/java/com/islamversity/reyan/MainActivity.kt]: It's for testing?

File: settings/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="surah_calligraphy">Surah Calligraphy</string>
+    <string name="aya_calligraphy">Aya Calligraphy</string>
```
Comment (rezaiyan) [inline settings/src/main/res/values/strings.xml:4]: I think `Calligraphy` is a technical term

**Review by rezaiyan — APPROVED** [review]
Well done

File: settings/src/main/res/values/strings.xml (hunk 3 above)
Comment (helloseyedjafari) [reply settings/src/main/res/values/strings.xml:4]: Let's do it when the designs arrive

File: app/src/main/java/com/islamversity/reyan/MainActivity.kt (hunk 2 above)
Comment (helloseyedjafari) [reply app/src/main/java/com/islamversity/reyan/MainActivity.kt]: nice catch
removed

**Pull request merged**